recipes/orange3-chem/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "Orange3-Chemoinformatics" %}
+{% set version = "0.0.2" %}
+{% set sha256 = "b29f3cbab8881b049a43ba44ed2fed68f9e1f6862496199d884091052e6a3d1d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - orange3 >=3.4.3
+    - pandas
+    - numpy
+    - scipy >=0.17
+    - rdkit
```

Comment:
Not declared in `setup.py`, though it is used.

Reply:
RDKit package it's not pip-installable, README specifies to install it before installing the add-on